tka/tka.go

```diff
@@ -543,3 +546,18 @@ func (a *Authority) Inform(updates []AUM) error {
 	a.state = c.state
 	return nil
 }
+
+// NodePermitted returns a nil error if the node key is authorized by the tailnet key
```

Comment:
Same, no need to spell out that a nil error is the success condition, for simple functions that only return an error.

`NodePermitted checks that nodeKeySignature authorizes nodeKey to be a member of the tailnet.`